File: Q01200-Q01299/Q01295_EASY/Q01295_Find_Numbers_with_Even_Number_of_Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int counter = 0, length;
        for(int i = 0; i < nums.length; i++){
            length = Integer.toString(nums[i]).length();
            if(length % 2 == 0){
                counter++;
            }        
        }
        return counter;
    }
}

File: Q01200-Q01299/Q01295_EASY/Q01295_Find_Numbers_with_Even_Number_of_Digits.py
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        counter = 0
        for i in range(len(nums)):
            length = len(str(nums[i]))
            if length % 2 == 0:
                counter += 1
        return counter